docs/html.md

```diff
@@ -8,238 +8,1053 @@ sidebar_position: 1
+- `<sup>` - Superscript text
+- `<big>` - Big text (1 unit of base text size)
+- `<small>` - Smaller text
+- `<tt>` - browser's default monospace text
+
+```html
+<!--Text in Bold-->
+<p><b>Bold Text</b></p>
+
+<!--Text in Strong-->
+<p><strong> Strong Text</strong></p>
+
+<!--Text in Italics-->
+<p><i>The Text inside italic Tag</i></p>
+
+<!--Text in Emphasize-->
+<p><em>Emphasized Text</em></p>
+
+<!-- small text -->
+<small>The text inside small Tag</small>
+
+<!-- Big text-->
+<big>The text inside big Tag</big>
+
+<!--Text in Superscript-->
+superscript <sup>Text</sup>
+
+<!--Text in Subscript-->
+subscript<sub>Text</sub>
+
+<!--Deleting andText in Insert-->
+<p>
+  The TajMahal is located in
+  <del>Bombay</del>
+  <ins>Agra</ins>
+</p>
+
+<!--Text in Highlight-->
+<mark>Highlighted Text</mark>
+
+<!-- monospace text -->
+<tt>Monospace text</tt>
+
+<!-- underline text -->
+<u>underline text</u>
+```
 
 </details>
 
 <details >
 <summary>
 Explain the usage of `<!DOCTYPE>` in HTML.
 </summary>
-                    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Laboriosam fuga qui nesciunt ipsa eius laborum repudiandae aut dignissimos voluptatum, perspiciatis molestias numquam iste, dolores veniam. Aliquam, pariatur dolore quia non explicabo eius! Pariatur omnis quam quo molestias sequi, non quod consequatur beatae rem blanditiis inventore ratione quisquam aspernatur facere vel?
+
+### `<!DOCTYPE>`
+
+tag is used to inform the browser about the version of HTML used in the document. It is called as the document type declaration (DTD). Technically <! DOCTYPE > is not a tag/element, it just an instruction to the browser about the document type.
 
 </details>
 
 <details >
 <summary>
 What is the significance of the `<head>` and `<body>` tag?
 </summary>
-                    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Laboriosam fuga qui nesciunt ipsa eius laborum repudiandae aut dignissimos voluptatum, perspiciatis molestias numquam iste, dolores veniam. Aliquam, pariatur dolore quia non explicabo eius! Pariatur omnis quam quo molestias sequi, non quod consequatur beatae rem blanditiis inventore ratione quisquam aspernatur facere vel?
+
+## `<head>` tag:
+
+The head tag in HTML is used to contain the metadata or information related to the document. It holds some of the most important tags like `<title>` , `<meta>` , and `<link>`.
+
+- From browser perspective, it is not mandatory to include a `<head>` tag inside the HTML document but in previous versions(4.0.1) it was mandatory to include it.
+
+- The tags like `<title>`, `<meta>` or `<link>` which are generally contained inside head will also work fine without the `<head>` tag or outside the `<head>` tag.
+
+- From the developer’s perspective, it is good to include the `<head>` tag inside the document because this syntax is widely used and it also gives a good structure to the document. Later this will help us to interact with the DOM elements in a structured way.
```

Comment:
`head` tag is used to define all the meta tag. write this also. 
Search other uses of head tag